test/performance/README.md

```diff
@@ -1,128 +1,189 @@
 
+   ```bash
+   $ cd $WORK_DIR
+   $ go run cmd/clusterloader.go --testconfig=./testing/load/performance-test-config.yaml --kubeconfig=<path>/wds-kubeconfig --provider=ks --v=2
+   ```
 
-3. Run your first workload:
 
 
-Set ``--kubeconfig`` flag to the path of the kubeconfig file of your cluster (e.g., WDS kubeconfig) and use the newly added Kubestellar provider
+5. Clean up the generated workload Kubernetes API objects from your cluster:
 
-```bash
-$ cd perf-tests/clusterloader2/
-$ go run cmd/clusterloader.go --testconfig=./testing/load/performance-test-config.yaml --kubeconfig=<path>/wds-kubeconfig --provider=ks --v=2
-```
+   ```bash
+   $ cd $WORK_DIR
+   $ ./cleanup.sh
+   ```
 
 
-## Collect the measurements 
+### b) Collect the creation & update timestamps for workload benchmark (TO BE REVISED: WORK-IN-PROGRESS)
 
-To collect the creation and status update for a k8s object proceed as following:  
+To collect the creation and status update for the workload benchmark objects proceed as following:  
 
 1. Create a kubeconfig with contexts for the `wds1`, `its1` and `wec1`:
 
-For example: 
+   For example: 
 
-```bash
-$ kubectl config get-contexts
-CURRENT   NAME     CLUSTER                                          AUTHINFO                                                                 NAMESPACE
-          its1     its1-cluster                                     its1-admin                                                                default
-          wec1     <url>:port#                                      <defaul-value>                                                            default
-*         wds1     wds1-cluster                                     wds1-admin                                                                default
-```
+   ```bash
+   $ kubectl config get-contexts
+   CURRENT   NAME     CLUSTER                                          AUTHINFO                                                                 NAMESPACE
+            its1     its1-cluster                                     its1-admin                                                                default
+            wec1     <url>:port#                                      <defaul-value>                                                            default
+   *         wds1     wds1-cluster                                     wds1-admin                                                                default
+   ```
 
-Use the following command to rename the default context name for `wec1` workload execution cluster:
+   Use the following command to rename the default context name for `wec1` workload execution cluster:
 
-```bash 
-$ kubectl config rename-context <default-wec1-context-name> wec1
-```
+   ```bash 
+   $ kubectl config rename-context <default-wec1-context-name> wec1
+   ```
 
 2. Configure the following parameters in the `main` function in the `metrics_collectory.py` script
```

Comment:
Why is the reader being told to "configure" something that already exists (https://github.com/kubestellar/kubestellar/blob/7f3736c3942530838da23d1bbab5444c98219144/test/performance/metrics_collector.py#L110-L117)?

Reply:
Removing the context of that subsection for now as it will be completely revised in a NEW PR, as already stated on my comment above.